lib/ansible/plugins/action/pause.py

```diff
@@ -169,7 +170,8 @@ def run(self, tmp=None, task_vars=None):
                     if not seconds:
                         new_settings = termios.tcgetattr(fd)
                         new_settings[0] = new_settings[0] | termios.ICRNL
-                        new_settings[3] = new_settings[3] | termios.ICANON
+                        if seconds or 'prompt' in self._task.args:
```

Comment:
I'm not sure the `if seconds` makes sense here since we'll never get to this point if `seconds` is defined.

Reply:
removed